Create ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md (#2638)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-3-0-get-context-dashboards-fix-7570-graft-release-cd-7573.md

---
date: ""

> title: Fix retrieving device context dashboards

Reviewer: ```suggestion
title: Device dashboards displayed for all devices
```

> title: Get context dashboards fix (#7570) [GRAFT][release/cd] (#7573)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fix retrieving context dashboards
```

> title: Get context dashboards fix (#7570) [GRAFT][release/cd] (#7573)

Reviewer 2: ```suggestion
title: Fix retrieving device context dashboards
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61691
version: 1021.3.0
---

> Get context dashboards fix (#7570) [GRAFT][release/cd] (#7573)

Author: The Claude AI suggested to use the following body:
```suggestion
In certain situations, retrieving context dashboards did not work as expected. This has now been fixed. Context dashboards can now be retrieved reliably in all cases. This fix ensures that users can always access and view their context dashboards as intended.
```

> Get context dashboards fix (#7570) [GRAFT][release/cd] (#7573)

Reviewer 2: Retrieving device context dashboards did not work as expected if device had space in it's name, so it was impossible to view device view as dashboard is by default first route for device in Cockpit. This has now been fixed.

> Get context dashboards fix (#7570) [GRAFT][release/cd] (#7573)

Reviewer 2: ```suggestion
Retrieving device context dashboards did not work as expected if device had space in it's name, so it was impossible to view device view as dashboard is by default first route for device in Cockpit. This has now been fixed.
```

> Retrieving device context dashboards did not work as expected if device had space in it's name, so it was impossible to view device view as dashboard is by default first route for device in Cockpit. This has now been fixed.

Reviewer: ```suggestion
Retrieving the device context dashboards did not work as expected if the device had a space in its name. As a result, it was impossible to view such devices as the dashboard is by default the initial view for devices in the Cockpit application. This issue has been fixed and the device dashboard is always displayed properly.
```

> Retrieving the device context dashboards did not work as expected if the device had a space in its name. As a result, it was impossible to view such devices as the dashboard is by default the initial view for devices in the Cockpit application. This issue has been fixed and the device dashboard is always displayed.

Reviewer 2: ```suggestion
Retrieving the device context dashboards did not work as expected if the device had a space in its type. As a result, it was impossible to view such devices as the dashboard is by default the initial view for devices in the Cockpit application. This issue has been fixed and the device dashboard is always displayed.
```

> Retrieving the device context dashboards did not work as expected if the device had a space in its name. As a result, it was impossible to view such devices as the dashboard is by default the initial view for devices in the Cockpit application. This issue has been fixed and the device dashboard is always displayed properly.

Reviewer: ```suggestion
Retrieving the device context dashboards did not work as expected if the device had a space in its name. As a result, it was impossible to view such devices as the dashboard is by default the initial view for devices in the Cockpit application. This issue has been fixed and the device dashboard is always displayed.
```
